bitcoin/Cargo.toml

```diff
@@ -23,6 +23,7 @@ serde = ["actual-serde", "hashes/serde", "internals/serde", "primitives/serde",
 secp-lowmemory = ["secp256k1/lowmemory"]
 secp-recovery = ["secp256k1/recovery"]
 bitcoinconsensus-std = ["bitcoinconsensus/std", "std"]
+arbitrary = []
```

Comment:
This is not needed

Reply:
right, if it's optional then this can be removed, thanks.